api/client/build.go

```diff
@@ -302,7 +302,7 @@ func (cli *DockerCli) CmdBuild(args ...string) error {
 		if jerr.Code == 0 {
 			jerr.Code = 1
 		}
-		return &utils.StatusError{Status: jerr.Message, StatusCode: jerr.Code}
+		return &StatusError{Status: jerr.Message, StatusCode: jerr.Code}
```

Comment:
Btw makes sense to make it value, not pointer. It isn't big deal for client though, just matter of Go-style. You can create issue for newbie contributors about it.